include/aws/common/private/lookup3.c

```diff
@@ -682,12 +697,16 @@ static uint32_t hashbig( const void *key, size_t length, uint32_t initval)
      * "k[2]<<8" actually reads beyond the end of the string, but
      * then shifts out the part it's not allowed to read.  Because the
      * string is aligned, the illegal read is in the same word as the
-     * rest of the string.  Every machine with memory protection I've seen
-     * does it on word boundaries, so is OK with this.  But VALGRIND will
-     * still catch it and complain.  The masking trick does make the hash
-     * noticably faster for short strings (like English words).
+     * rest of the string. Every machine with memory protection I've seen
+     * does it on word boundaries, so is OK with this. But VALGRIND and CBMC
+     * will still catch it and complain. CBMC will ignore this type of error
+     * in the code block between the pragmas "CPROVER check push" and
+     * "CPROVER check pop". The masking trick does make the hash noticably
+     * faster for short strings (like English words).
      */
 #ifndef VALGRIND
+#pragma CPROVER check push
```

Comment:
+1, this needs a guard or it will fail CI

Reply:
done